backend/src/main/java/com/engineering/thesis/backend/controller/AppointmentController.java

```diff
@@ -18,8 +20,13 @@ public void create(@RequestBody Appointment appointment) {
         appointmentService.create(appointment);
     }
 
+    @PutMapping("/update")
+    public void update(@RequestBody Appointment appointment) {
```

Comment:
I think it's fine to have update without id parm (opposite to User EP). It's still doing job just ok.